docs/src/python/user-guide/io/database.py

```diff
@@ -1,4 +1,4 @@
-"""
+f"""
```

Comment:
This should be a regular multi line string, not an f-string

Reply:
Change made